src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -2091,6 +2091,27 @@ public void DrawImage(Image image, Point[] destPoints, Rectangle srcRect, Graphi
             CheckErrorStatus(status);
         }
 
+        /// <summary>
+        /// Draws the image stored in the a <see cref="CachedBitmap"/> object.
+        /// </summary>
+        /// <param name="cachedBitmap">The <see cref="CachedBitmap"/> that contains the image to be drawn.</param>
+        /// <param name="x">The x-coordinate of the upper-left corner of the drawn image.</param>
+        /// <param name="y">The y-coordinate of the upper-left corner of the drawn image.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="cachedBitmap"/> is <see langword="null"/>.</exception>
+        /// <exception cref="PlatformNotSupportedException">The current operating system is not Windows.</exception>
```

Comment:
...or libgdiplus 6.1 or higher?

Reply:
Whoops, this was left over from when I didn't plan on implementing the non-Windows support.